eclair-core/src/test/scala/fr/acinq/eclair/channel/states/a/WaitForOpenChannelStateSpec.scala

```diff
@@ -216,6 +216,31 @@ class WaitForOpenChannelStateSpec extends TestKitBaseClass with FixtureAnyFunSui
     awaitCond(bob.stateName == WAIT_FOR_FUNDING_CREATED)
   }
 
+  test("recv OpenChannel (upfront shutdown script)", Tag(StateTestsTags.OptionUpfrontShutdownScript)) { f =>
+    import f._
+    val open = alice2bob.expectMsgType[OpenChannel]
```

Comment:
I would test the contents of the tlv here as well, otherwise if no script is sent your test still passes:
```suggestion
    val open = alice2bob.expectMsgType[OpenChannel]
    assert(open.upfrontShutdownScript_opt === Some(Alice.channelParams.defaultFinalScriptPubKey))
```

Reply:
done in https://github.com/ACINQ/eclair/pull/1846/commits/e3174057030eb52428df1f77937f56568db4dfe4